test/prepare.bats

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bats
+
+@test "Test scripts directory created, if it does not exist" {
+    rm -rf scripts/
+    run "src/prepare.sh"
+    [ -e scripts/ ]
+    rm -rf scripts/
+}
+
+@test "Test go chaincode was downloaded" {
+    rm -rf scripts/
+    run "src/prepare.sh"
+    [ -f scripts/router.sh ]
+    [ -d scripts/go-chaincode ]
+    [ -f scripts/go-chaincode/build.sh ]
+    [ -f scripts/go-chaincode/test.sh ]
+    [ -f scripts/go-chaincode/deploy.sh ]
```

Comment:
What if it created additional directories?

Reply:
Again,  just like the prepare.sh and prepare-unstable.sh,  we are going to rely on you to create those scripts and the tests for them.   We just added enough to get thins moving.